src/OrchardCore/OrchardCore.ResourceManagement/TagHelpers/StyleTagHelper.cs

```diff
@@ -200,12 +200,12 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
 
             if (!string.IsNullOrEmpty(Culture))
             {
-                definition.SetCultures(Culture.Split(',', StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries));
+                definition.SetCultures(Culture.Split(_splitSeparators, StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries));
```

Comment:
Didn't we discuss today that since this is part of the `Culture` parameter, not splitting parameters, it shouldn't be used here?

Reply:
Yes. But when I looked at ScriptTagHelper, we are actually using both separators for culture. So I used both to ensure the logic is consistent on style and script.